tests/conftest.py

```diff
@@ -145,7 +145,7 @@ def build(self, **kwargs):
             # Get event from queryset so it has the start_time annotation from the manager
             metro_event = LAMetroEvent.objects.get(id=event.id)
 
-            if metro_event.start_time < datetime.now(timezone.utc):
+            if has_broadcast:
```

Comment:
It'd be more intuitive for this fixture to create the broadcast for past events unless has broadcast is False. Want to try that revision?